buildspec_pr.yaml

```diff
@@ -0,0 +1,13 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - pip install setuptools wheel twine
+      - python setup.py sdist
+      - pip install tox
```

Comment:
Can this be in the first `pip` command?

Reply:
good call, i'll give that a try, but am unsure whether there's motivation behind the current structure which I reused.

Reply:
verified that it can. updated the PR